packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1093,6 +1100,12 @@ class DropdownButton<T> extends StatefulWidget {
   /// instead.
   final Color? dropdownColor;
 
+  /// The maximum height of the menu.
+  ///
+  /// The [menuMaxHeight] should be less than or equal to the maximum height of
+  /// one less row, otherwise it is invalid.
```

Comment:
Could you clarify what you mean here by one less row? Does this mean that if you have a menu with 5 items, it can be the maximum height of the height of 4 items? Why not up to 5 items (maybe a height that shows half an item)?

Reply:
Ahh, you mean one less row compared to the view itself. Maybe clarify that, because my first thought was one less row relative to the maximum height that the menu can be, assuming the entire menu fit on the screen.